std/random.d

```diff
@@ -68,7 +68,8 @@ import std.traits;
 
     // Generate a uniformly-distributed real in the range [0, 100)
     auto r = uniform(0.0L, 100.0L, rnd);
-    assert(r == 79.65429843861011285);
+    import std.math : approxEqual;
+    assert(approxEqual(r, 79.65429843861011285));
 
```

Comment:
Give @WebDrake advice, I think it's better 5o simply remove the equality comparisons and instead assert that the  numbers are in the expected intervals.

Reply:
@ZombineDev I take it you're talking here only about the floating-point number?  The integer variates generated elsewhere in the unittest should be exactly equal, assuming that `Random` is reliably `Mt19937`.

Reply:
Yes, I mean the integer cases too, since I don't think knowledge of the Random's implementation should be assumed for the readers of the first example in the documentation. Testing the exact values should be done only when the exact random engine is explicitly specified, in addition to its seed, IMO.

Reply:
Oh, so you mean in this case you'd check that the resulting number is `>= 0` and `< 100`, and for the `uniform(0, 15, rnd)` call you'd check that it's `>= 0` and `< 15`?

That sounds reasonable, yes.  Note that for the `uniform!uint` case this won't really work since the range of possible values covers the entire range of possible values for that type; but you don't really have to validate that result at all anyway.

Reply:
Done.